practice/src/javascripts/services/baseService.js

```diff
@@ -0,0 +1,14 @@
+import { APIHelper } from '../helpers/api';
+
+export default class BaseService {
+  constructor(endpoint) {
+    this.endpoint = endpoint;
+  }
+
+  /**
+   * Fetch all products from the API
+   */
+  async getAllProducts() {
```

Comment:
This is a base service common, so you can use it for multiple objects such as users, companies, products, etc. Therefore, don't use a specific name for an object in the common class.